lib/components/Input/base/base.scss

```diff
@@ -298,6 +298,12 @@
                 border-radius: var(--component-field-border-radius-md);
                 gap: var(--core-spacing-800);
             }
+            &--lg {
+                padding-inline: var(--component-field-spacing-padding-md);
+                height: var(--core-spacing-2000);
+                border-radius: var(--component-field-border-radius-md);
+                gap: var(--component-field-spacing-gap-sm); 
+            }
```

Comment:
as checked the figma, newly added size is `md`. previous `md` has to changed to `lg` size. please check